tests/pkg_fatfs_vfs/Makefile

```diff
@@ -43,8 +43,10 @@ BOARD_WHITELIST := airfy-beacon arduino-due arduino-duemilanove arduino-mega2560
                    nz32-sc151 openmote-cc2538 pba-d-01-kw2x remote-pa remote-reva \
                    remote-revb samd21-xpro saml21-xpro samr21-xpro sltb001a \
                    sodaq-autonomo sodaq-explorer spark-core stm32f0discovery \
-                   stm32f3discovery stm32f4discovery udoo waspmote-pro \
-                   yunjia-nrf51822 native
+                   stm32f3discovery stm32f4discovery telosb udoo waspmote-pro \
```

Comment:
These BOARD_WHITELIST changes look unrelated, please remove them, you can directly squash as they are easy to review.

Reply:
Yup, those were a rebase mistake.